_posts/2021-12-13-log4j-issue.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: CVE-2021-44228 ("Log4Shell") Assessment
+categories: blog
+---
+
+Major news carriers have been reporting recently on log4j vulnerabilities in
+Java applications. Both the OME team in Dundee as well as Glencoe Software have
+evaluated the libraries used by OMERO.server and OMERO.insight. We can say with
+confidence that OMERO and OMERO Plus are not vulnerable, as they do not utilize log4j
+and no immediate action that needs to be taken.
+
+The only remaining use of log4j in OME software is in the
+[deprecated](https://imagej.net/pipermail/imagej-devel/2015-March/002513.html)
+`loci_tools.jar` which we will now be removing. The `bioformats_package.jar` uber-jar
```

Comment:
To be discussed but can we be authoritative and say "which we will now be removing in the next release of Bio-Formats"? /cc @dgault @melissalinkert